app/components/search-facet-taxonomy.js

```diff
@@ -99,6 +100,20 @@ export default Ember.Component.extend(Analytics, {
             this._expand(item);
         }
     },
+    _expandDefault(){
```

Comment:
Minor:
Not sure what the es6 linting will suggest, but there is some inconsistent paren spacing. I think it is supposed to be `fun() {` instead of `fun(){` but I actually don't know which we want to do, either way consistency is better.